config-template.toml

```diff
@@ -413,6 +413,54 @@ rpc_endpoints = ["wss://rpc.ibp.network/paseo"]
 testnet = true
 verifier = "novasama"
 
+[[inject_chains]]
+name = "asset-hub-paseo"
```

Comment:
asset hub is already added, so you can't use it here. As well as bridge hub, people chain.